packages/wallet/src/redux/protocols/indirect-defunding/readme.md

```diff
@@ -0,0 +1,44 @@
+# Indirect De-Funding Protocol
+
+The purpose of this protocol is handle de-funding a channel that has been indirectly funded.
+
+It covers:
+
+- Checking that a channel is closed (either finalized on chain or a conclusion proof exists)
+- Crafting a ledger update that allocates the funds to the players.
+- Waiting for a ledger response from the opponent.
+
+## State machine
+
+### Player A State machine
+
+```mermaid
+graph TD
+  St((start))-->WFAp(WaitForApproval)
```

Comment:
Probably don't need this, as we can assume that approval has already been given.